Automatic/move-images-to-other-tab.ini

```diff
@@ -1,7 +1,16 @@
 [Command]
 Name=Move Images to Other Tab
-Input=image/png
 Automatic=true
-Remove=true
 Icon=\xf03e
 Tab=&Images
+Remove=true
```

Comment:
"Remove" is not needed here. The `setData(mimeOutputTab, imageTab)` changes the default tab for storing the new clipboard content. "Remove" would also avoid running other automatic commands which could be unexpected.